nat/service_pfctl.go

```diff
@@ -40,15 +40,11 @@ func (service *servicePFCtl) Add(rule RuleForwarding) {
 func (service *servicePFCtl) Start() error {
 	err := service.ipForward.Enable()
 	if err != nil {
-		return err
+		log.Warn(natLogPrefix, "Failed to enable IP forwarding: ", err)
```

Comment:
Now it's double loging, is not it?

Reply:
yeah, it prints messages from both functions.

Reply:
I think we can leave it this way for the release.